docs/adr/adr_0016.md

```diff
@@ -20,14 +21,21 @@ This ADR should have been written prior to implementation. But we started docume
 
 ## Context
 
-What is the issue that we’re seeing that is motivating this decision
-or change.
+_secureCodeBox_ needs to saves two things during the execution of a scan: the "raw results / output" of the scanner (e.g. the XML output of nmap) and the findings the parsers extract from it. In the v1 of the secureCodeBox these were stored in camunda variables which were in turn stored in the underlying database as blobs. As both the raw results and findings can get big (multiple megabytes) this requiered a lot of maintenance work to keep the database working correctly without overflowing.
 
 ## Decision
 
-What is the change that we’re actually proposing or doing.
+In v2 we decided to shift the storage to store the raw results and findings as separate files in an S3 bucket.
+
+S3 was chosen because:
+
+    - it can handle large files without much problems
+    - is relatively easy to use
+    - is well supported by (most) cloud supporters
+    - can be "substituted locally" by minio
+    - can be used with presigned urls to easily give individual components temporary read or write access to specific files in the s3 bucket
```

Comment:
```suggestion
- it can handle large files without much problems
- is relatively easy to use
- is well supported by (most) cloud supporters
- can be "substituted locally" by minio
- can be used with presigned urls to easily give individual components temporary read or write access to specific files in the s3 bucket
```